python-main.js

```diff
@@ -1164,9 +1164,13 @@ function web_editor(config) {
         });
         $("#command-help").click(function (e) {
             // Show help
+            if($(window).width() > 620){
             $(".helpsupport_container").css("top", $("#command-help").offset().top + $("#toolbox").height() + 10);
             $(".helpsupport_container").css("left", $("#command-help").offset().left);
-
+            }else{
+                $(".helpsupport_container").css("top", $("#command-help").offset().top + $("#toolbox").height() + 10);
+                $(".helpsupport_container").css("left", $("#command-save").offset().left);
```

Comment:
Is this meant to be `#command-save`?
If it is, then it means that changing the save button would affect this pop as well. Is there anyway we can bind the calculation to only `#command-help`?

Reply:
Done